lib/api_browser/app/views/controller.html

```diff
@@ -42,9 +42,13 @@ <h1 class="page-header">
     </div>
     </div>
   </div>
-  <div ng-if="controller.media_type" ng-controller="TypeCtrl">
+  <div ng-if="controller.media_type && controller.media_type.family != 'string'" ng-controller="TypeCtrl">
```

Comment:
I hesitated to do that because I think that view is kind of suited for any “type” …right?
Also what we need to display in the other case, is simply the string of the mediatype identifier and felt, that it was a controller.html concern, not a “type” concern to know about special mediatypes and how to display them